frontend/src/index.jsx

```diff
@@ -3,7 +3,7 @@ import { BrowserRouter } from 'react-router-dom'
 import TimeAgo from 'javascript-time-ago'
 import en from 'javascript-time-ago/locale/en.json'
 import App from './App'
-import '@fontsource/roboto'
+import '@fontsource/poppins'
```

Comment:
eslint failing for no apparent reason? ignore?

Reply:
I think if there's no clear reason for eslint to be failing, then happy to put an ignore and/or flag for investigation down the line if it happens again.